### PR TITLE
Added logo (icon+text) to the readme

### PR DESCRIPTION
**Pull request by Sharps**
I saw that you needed a logo so I thought I'd design something. This is my first pull request but I found this hackathon thing and figured it's a good time to try and learn. 
The logo should probably be in the repo itself but I wasn't sure how I'd link it in the readme then, so I just uploaded it to imgur for now. It's basically supposed to be the three nuget dots within a shield, if you think that's not generic enough I could do something without the dots too I guess.

### CODE REVIEW

**Comment by digitalcoyote** [comment]
### Thanks for the PR!

I love the style, I was actually playing around in inkscape with a similar design in mind. Once the image is pushed to the git repo, you can use https://raw.githack.com/ to get a cdn url for it (helps to take the load off of Github's servers and is the 'recommended' way of doing it as far as i Know). I'll end up needing it in the [Docs](https://github.com/digitalcoyote/NuGetDefenseDocs) as well (I can handle this, or if you need another PR, I can explain how to do it). 

TL;DR - If you can add it to the repo as part of this PR, I'll merge it in. Great Work!

**Comment by Sharps** [comment]
Added logo to images folder and linked the images to that. Hope that works.
